packages/geolocation/src/utils/logger.js

```diff
@@ -3,20 +3,12 @@
 const { format, createLogger, transports } = require('winston');
 const { combine, timestamp, printf } = format;
 
```

Comment:
This file is not longer need it

Reply:
Can you check it, did I make the changes?

Reply:
That part is fine now